lmcache/v1/compute/models/utils.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2024-2025 LMCache Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Standard
+from typing import Dict
+
+# Third Party
+from torch import nn
+
+# First Party
+from lmcache.logging import init_logger
+
+logger = init_logger(__name__)
+
+
+# TODO (Jiayi): Need to infer the model type from vllm model
+
+
+def infer_model_from_vllm(vllm_model, blender):
+    model_name = type(vllm_model).__name__
+    if model_name == "LlamaForCausalLM":
+        # First Party
+        from lmcache.v1.compute.models.llama import LMCLlamaModel
+
+        return LMCLlamaModel(vllm_model, blender)
+    else:
+        # TODO(Jiayi): Add support for more models
+        raise NotImplementedError(
+            f"Model type {model_name} is not supported in LMCache."
+        )
+
+
+class VLLMModelTracker:
+    _vllm_models: Dict[str, nn.Module] = {}
+
+    @classmethod
+    def register_model(
```

Comment:
I didn't find anywhere call this method, I guess this is why i encountered `vllm model for vllm-instance not found`, so is the caller out of this PR?